components/src/core/Chip/Chip.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { MD3Theme, Chip as PaperChip, ChipProps as PaperChipProps, useTheme } from 'react-native-paper';
+import { $DeepPartial } from '@callstack/react-theme-provider';
+import { TextStyle } from 'react-native';
+import { Icon } from '../Icon';
+import { IconSource } from '../__types__';
+
+type ChipStyles = {
+    label?: TextStyle;
+};
+
+const makeStyles = (): ChipStyles => ({
```

Comment:
No need to define makeStyles, as it is only getting used once.
You can directly set `fontFamily` in textStyles.